src/Form.js

```diff
@@ -0,0 +1,45 @@
+import firebase from './firebase.js';
+import { useState } from 'react';
+
+const Form = () => {
+    const [ taskName, setTask ] = useState('');
+    const handleOnChange = (e) => {
+        setTask(e.target.value);
+    }
+
+    const addTask = () => {
+        const taskRef = firebase.database().ref('Task');
+        const task = {
+            taskName,
```

Comment:
I think this might need to be a key-value pair? Right now it's just I think a value?